### DISCUSSION TITLE
UI for Light Group

### DISCUSSION THREAD

**Discussion by pw4056** [home-assistant/frontend#7159] [Configuration] [answered]
## The request
Request an UI-editor for light groups, as an alternative to YAML.
https://www.home-assistant.io/integrations/light.group/

## The alternatives
YAML is the alternative way.

## Additional information
Maybe this is an idea also for other groups, like notify and cover.

**Comment by iantrich** [comment]
@bramkragten does this make more sense as a helper or should this integration just implement config flow?

**Reply by bramkragten** [reply]
It should not be a config flow, as it doesn't communicate with a device or service. I would like a new section next to helpers, groups. That would cover light, switch, and normal groups.

**Comment by maverick1872** [comment]
I would be interested in helping with this implementation if need be. Is there maybe some reference material or documentation that can point me in the right direction for a starting point?

**Reply by bramkragten** [reply]
Look at the code for the input helpers, this should be similar. We first need support in the backend, with a collection helper and the websockets endpoints.

**Comment by KTibow** [comment]
Possible duplicate of #7483

**Reply by maverick1872** [reply]
I agree - although there has been no traction on the other discussion. I would think it would make sense to close that one, and consolidate here.

**Reply by KTibow** [reply]
Duplicate of #7159, which was transferred to Discussions, but previously wasn't (so I didn't know that it existed).

**Reply by Mincka** [reply]
#7483 was not limited to **light** groups so maybe this thread title should be also edited

**Comment by leroyboerefijn** [comment]
Is this something that will be worked on in the near future? Since home assistant is transfered to the new 'stable' version numbering, I would expect that it is a priority to move away from yaml for at least the basic functions. I consider having a group with multiple light bulbs that are in the same housing a basic functionality, since you often want to change brightness/color_temp/rgb of the entire lamp.

**Reply by zsarnett** [reply]
Where did you hear about a Stable Version number? Its just a different Version Numbering. This will be done when a dev wants to pick it up and do it. But I am sure this is something in the timeline of the nearest of futures

**Reply by leroyboerefijn** [reply]
I might have misunderstood then. There was a lot of talk about version 1.0, which then became a different version numbering. I assumed this was still intended to be the stable version.

**Reply by iantrich** [reply]
No, the 1.0 beta was an homage to the work we've done to get where we are. It will continue to evolve as it has always.

**Comment by lzinga** [comment]
I would like to re-iterate that this discussion is specifically about lights, I believe the scope of this should be expanded to the larger `entities` list as a whole. I believe it would be beneficial to rename the discussion in an effort to better track `groups` as a whole in home assistant frontend.

As a pre-cursor to manual creation of groups depending on the difficulty it may be easier at the beginning to simply group all entities by their `Integration`.

**Comment by maverick1872** [comment]
I'm gonna take a stab at implementing the necessary feature set for this one over the holidays. There's a lot I'm gonna have to familiarize myself with to even start though. If anyone here has an interest in giving me some pointers about where to start I'd love the insight though.

**Comment by W-Floyd** [comment, marked as answer]
This has been available since early last year 😊
